modules/core/src/main/scala/sup/HealthResult.scala

```diff
@@ -11,25 +10,21 @@ import sup.data.Tagged
 
 final case class HealthResult[H[_]](value: H[Health]) extends AnyVal {
   def transform[I[_]](f: H[Health] => I[Health]): HealthResult[I] = HealthResult(f(value))
+
+  def mapK[G[_]](fg: H ~> G): HealthResult[G] = new HealthResult[G](fg(value))
 }
 
-object HealthResult {
+object HealthResult extends HealthResultTaglessInstances {
   def const[H[_]: Applicative](health: Health): HealthResult[H] = HealthResult(health.pure[H])
 
   val one: Health => HealthResult[Id] = HealthResult[Id]
   def tagged[Tag](tag: Tag, head: Health): HealthResult[Tagged[Tag, *]] = HealthResult(Tagged(tag, head))
 
-  implicit val functorK: FunctorK[HealthResult] = new FunctorK[HealthResult] {
-    def mapK[F[_], G[_]](hf: HealthResult[F])(fg: F ~> G): HealthResult[G] = new HealthResult[G](fg(hf.value))
-  }
-
   implicit def monoid[H[_]: Applicative](implicit M: Monoid[Health]): Monoid[HealthResult[H]] =
-    new Monoid[HealthResult[H]] {
-      override val empty: HealthResult[H] = HealthResult.const[H](M.empty)
-
-      override def combine(x: HealthResult[H], y: HealthResult[H]): HealthResult[H] =
-        HealthResult(Applicative.monoid[H, Health].combine(x.value, y.value))
-    }
+    Monoid.instance(
```

Comment:
I had some weird error on Scala complaining about a generated bridge being in conflict after type erasure, but luckily, `Monoid.instance` is a good drop-in replacement.